### DISCUSSION TITLE
How to export a backup from Point-in-Time into another project?

### DISCUSSION THREAD

**Discussion by guilimasp** [supabase/supabase#27070] [Questions]
I have Point-in-Time turned on for a certain project. Project is called "Prod" and contains all of our prod tables.

I need restore a backup from "Prod" from a few days ago. The backup is available. However, I want to restore it into a different project ("Temp"). I cannot lose "Prod".

Supabase gives me no interface option to do so. Is there anyway I can do that?

**Comment by amusakunov** [comment]
Would be nice to get an answer for this one @supabase team

**Comment by theranajayant** [comment]
Searching for the same. Did you get any solution!

**Comment by julian-gargicevich** [comment]
+1

These are the things I sometimes miss from bigger vendors like Azure or Aws, where moving around/copying/restoring databases is just a few clicks.

**Comment by trogers1** [comment]
Yeah this seems like a fairly common usecase (copying an old version of the DB to a new, temporary db to recover some data). I'm surprised this isn't supported. I hope the supabase team prioritizes this. Otherwise, I may have to move back to AWS or build backup system myself 😑

**Comment by michaelyork** [comment]
Big +1!

**Comment by castvoid** [comment]
Looks like this has recently landed: [Supabase Blog - Restore to a New Project](https://supabase.com/blog/restore-to-a-new-project)
